### PR TITLE
use an info.py file to allow integration with casa-distro/bv_maker

### PR DESCRIPTION
**Pull request by denisri**
this is just a separation of what was in setup.py into 2 files, the
info.py file being read by bv_maker not only for setup. Moreover it
allows to get info at runtime (to get library version etc) when needed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
+setuptools.setup(
+    name=release_info['NAME'],
+    version=release_info['__version__'],
+    description=release_info['DESCRIPTION'],
+    author=release_info['AUTHOR'],
+    author_email=release_info['AUTHOR_EMAIL']
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url=release_info['URL'],
+    packages=['moving_averages', 'moving_averages.distance'],
```
Comment (mpascucci) [inline setup.py]: Joel a changé cette ligne en ajoutant 'moving_averages.isomap' au packages. Je ne sais pas qui de vous deux va merge en premier... l'autre en tout cas aura un merge conflict ici

**Review by mpascucci — DISMISSED** [review]

**Comment by mpascucci** [comment]
Could you please merge this into Dev branch instead of main?

**Comment by denisri** [comment]
Things have moved a lot in setup.py which I propose to move in a large part in info.py, thus the PR leads to conflicts that I can't solve using the interface. I'll try rebasing over dev.

**Comment by denisri** [comment]
(rebase done)

**Review by mpascucci — APPROVED** [review]

**Pull request merged**